spec/requests/custom_field_cache_spec.rb

```diff
@@ -94,36 +98,25 @@ def visit_cost_reports_index
     expect_filter_all_to_not_exist(custom_field)
   end
 
-  it 'allows for changing the db table between requests if no caching is done' do
-    old_table_name = WorkPackageCustomField.table_name
-    new_table_name = 'custom_fields_clone'
-    new_id = custom_field.id + 1
-
-    begin
-      mock_cache_classes_setting_with(false)
+  it 'allows for changing the db entries directly via SQL between requests \
+      if no caching is done (this could also mean switching dbs)' do
```

Comment:
I love those concise descriptions you find :D

> allows for changing the db entries directly via SQL between requests if no caching is done (this could also mean switching dbs; but after all we are only testing this for SaaS, where we have one ruby process serving multiple tenants... oh boy, but lets focus: This test is important!)

Just kidding! The name is fine :cat: